Create mockup index.html

matthewjchandler opened the pull request:
This is a proposed new DataSpace Help page. I wrote it from scratch, so the HTML is clean and simple. It references the default DSpace "styles.css" the same way as the vanilla "index.html."

matthewjchandler committed:
Create mockup index.html

This is a proposed new DataSpace Help page. I wrote it from scratch, so the HTML is clean and simple. It references the default DSpace "styles.css" the same way as the vanilla "index.html."

jrgriffiniii commented:
Thank you very much for this @matthewjchandler! I am on a Samvera community sprint this afternoon, but I can look to review this soon.

matthewjchandler committed:
Create mockup about.html

This is in essence what the new "About" page should look like. However, it is unclear how exactly the DSpace JSPUI constructs the "About" page, so this file may need to be revised--perhaps to include just the body. It is also unclear whether JSPUI will reference "styles.css," and if so, where that is in relation to this file's directory.

matthewjchandler committed:
Create mockup about-body

I suspect that the DSpace JSPUI takes the content of the "About" page as an option value inserted into the "Messages.properties" file. In that case, we don't want a full HTML page, as in the "about.html" file in the mockup folder here. Instead, we would just want the static content wrapped in <div></div> tags, so it can be easily dropped into the HTML page JSPUI automatically constructs for the viewer. I'm not certain about the stylesheets used in that case, but I believe what I have here will look acceptable regardless.
